zio/csvio/writer.go

```diff
@@ -41,6 +41,11 @@ func (w *Writer) Close() error {
 	return w.writer.Close()
 }
 
+func (w *Writer) Flush() error {
+	w.encoder.Flush()
+	return nil
```

Comment:
```suggestion
	return w.encoder.Error()
```

Reply:
Have to call csv.Writer.Error().  Will do.